### DISCUSSION TITLE
Moving On

### DISCUSSION THREAD

**Discussion by PJF9** [mrdbourke/pytorch-deep-learning#361] [Q&A]
First of all what an amazing course that was.
Thank you very much for this content, keep the hard work.
I have only one question (that does not apply to that course). So my question is the following: I want to start and master NLP, what courses/videon or resources in general fo you recommend me?
(or anyone who knows)

**Comment by mrdbourke** [comment]
Hi @PJF9,

Thank you for the kind words! I really appreciate it!

If you'd like to learn more about PyTorch/ML in general I'd recommend the resources here:

* PyTorch Extra Resources - https://www.learnpytorch.io/pytorch_extra_resources/ 

Otherwise, for NLP specifically, I'd go into Hugging Face's course for `transformers` (my favourite NLP-focused library). 

* Hugging Face course - https://huggingface.co/course/chapter1/1

**Comment by PJF9** [comment]
Thank you, very much!
